src/main/java/seedu/address/logic/commands/findcommand/FindTeamCommand.java

```diff
@@ -54,4 +58,18 @@ public CommandResult execute(Model model) {
         model.updateHistory(this);
         return new CommandResult(MESSAGE_SUCCESS, CommandType.T);
     }
+
+    @Override
+    public boolean equals(Object otherFindCommand) {
+        if (otherFindCommand == this) {
+            return true;
+        }
+
+        if (!(otherFindCommand instanceof FindTeamCommand)) {
+            return false;
+        }
+
+        return name.equals(((FindTeamCommand) otherFindCommand).name)
+                && projectName.equals(((FindTeamCommand) otherFindCommand).projectName);
```

Comment:
Maybe you can consider comparing subject name too

Reply:
Subject name isnt in this command tho